DataFormats/BTauReco/interface/DeepDoubleBTagInfo.h

```diff
@@ -0,0 +1,16 @@
+#ifndef DataFormats_BTauReco_DeepDoubleBTagInfo_h
+#define DataFormats_BTauReco_DeepDoubleBTagInfo_h
+
+#include "DataFormats/BTauReco/interface/FeaturesTagInfo.h"
+#include "DataFormats/BTauReco/interface/DeepDoubleXFeatures.h"
+
+//keeping for posterity
+namespace reco {
+
+typedef  FeaturesTagInfo<btagbtvdeep::DeepDoubleXFeatures> DeepDoubleBTagInfo;
```

Comment:
can this ```DeepDoubleBTagInfo``` really be used to read the old data?

Reply:
also, to be able to read DeepDoubleBTagInfo from the old data, I think that we still need an entry in the classes.h and classes_def.xml files.
Does this recovery of the previously removed file imply that we actually have some data with DeepDoubleBTagInfo persisted?

Reply:
The updated implementation doesn't look obviously correct.
The best solution is to make a file in the baseline with such data DeepDoubleBTagInfo and then try to read it in a job using code from this PR.

Reply:
another option is to use a reactive approach and postpone reintroduction of the bw compatibility file until the unreadable data shows up.
It is clear that DeepDoubleBTagInfo is not a part of the production outputs.
So, I suggest to drop this file from the PR.

Reply:
Alright, I checked against a test file produced in base. To be consistent it seems necessary to keep Features as well.